pyproject.toml

```diff
@@ -59,6 +59,8 @@ develop = [
     "isort>=5,<6",
     "ruamel.yaml",
     "ruff>=0.3,<0.4",
+    "mdformat",
```

Comment:
these probably need to be added to the `dev-environment.yml` conda env too @ngoldbaum